### ISSUE TITLE
Details of yaml file of training when train the model

### ISSUE THREAD

**Issue by connorye** [CDchenlin/DentalVLM#2]
Thanks for your great work. I am trying to use  provided details of the model to reproduce the article so that I can create alike model to adapted to my work.At first,I tried Open-groundingDINO, a open-sourced method to train G-DINO,but it seemed that  it was not ideal for my own work. Then i tried the mmdection version to train G-DINO and it improved a lot.However,I only get the lr and weight_decay in the article.I am wondering that more details could be provided when training the model.Thanks for any information could be provided.

**Comment by chenlindu** [comment]
Hello, thank you for your interest. We employ the exact same learning rate and weight decay parameters as those provided by the MMDetection framework. Hope it works.

**Comment by connorye** [comment]
Thanks for the provided information.
